<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeEditor</class>
 <widget class="QWidget" name="RangeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="hsvButton">
        <property name="text">
         <string notr="true">HSV</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rgbButton">
        <property name="text">
         <string notr="true">RGB</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="lowerGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string notr="true">LOWER</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QStackedWidget" name="lowerStackedWidget">
        <widget class="HsvRangeEditor" name="lowerHsvRangeEditor"/>
        <widget class="RgbRangeEditor" name="lowerRgbRangeEditor"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="upperGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string notr="true">UPPER</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QStackedWidget" name="upperStackedWidget">
        <widget class="HsvRangeEditor" name="upperHsvRangeEditor"/>
        <widget class="RgbRangeEditor" name="upperRgbRangeEditor"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string notr="true">...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RgbRangeEditor</class>
   <extends>QWidget</extends>
   <header>rangeEditors</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HsvRangeEditor</class>
   <extends>QWidget</extends>
   <header>rangeEditors</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
